Modules/Options/QuestieOptionsDefaults.lua

```diff
@@ -15,6 +15,7 @@ function QuestieOptionsDefaults:Load()
             monsterScale = 1,
             objectScale = 1,
             globalScale = 0.6,
+            globalTownsfolkScale = 0.6,
```

Comment:
You introduced this, but does not use it at any place.

Reply:
This is used in the `globalTownsfolkScale` in `QuestieOptionsIcons.lua` to set the default value to 0.6 on the slider.

Reply:
True. But it is not used to change the scaling of any icon.